src/cript/nodes/main.py

```diff
@@ -37,6 +39,10 @@ def __init__(self, *args, **kwargs):
             self.__dict__["validator_instance"] = cls(schema, resolver=resolver)
         else:
             self.__dict__["validator_instance"] = cls(schema)
+
+        # Early exit for initialized nodes
+        if self.initialized:
+            return
```

Comment:
you need to assign valid kwargs to self at this point and then return

Reply:
Did I do it right now?

Reply:
yes but we need to make sure not to include the extra fields that the api returns, you can refactor the `retrieve_by_uuid` and create a separate method from
```
allowed_data = {}
for key in data:
    if key in self.__dict__["schema"]["$defs"][f"{self.__class__.__name__}Post"]["properties"]:
        setattr(self, key, data[key])
        allowed_data[key] = data[key]
self.__dict__["__original__"] = copy.deepcopy(allowed_data)
```

Reply:
Thanks for the pointers.
I used that snippet to refactor it a little bit.

I think there is some room to streamline this a little bit.
But we can address that in a refactor down the line.